Corona-Warn-App/build.gradle

```diff
@@ -242,6 +242,9 @@ dependencies {
     // noinspection GradleDependency - needed for SDK 23 compatibility, in combination with com.journeyapps:zxing-android-embedded:4.1.0
     implementation 'com.google.zxing:core:3.3.0'
 
+    //ENA
+    implementation fileTree(dir: 'libs', include: ['*.aar', '*.jar'], exclude: [])
```

Comment:
imo, we should not include all aar or jar files but instead explicitly import the correct play services so that an upgrade is traceable in the build.gradle

Reply:
all right, I will make an explicit import for tracability.